### PR TITLE
feat: add dynamic unbound period

### LINKED ISSUE #973: Namadillo: Unbonding period in Unstake screen is hardcoded
The unbonding period is hardcoded as 21 days.
2024-08-07-120212_539x313_scrot

### PR DESCRIPTION
**Pull request by euharrison**
Closes #973 

Add dynamic unbound period

Screenshot 2024-08-09 at 14 50 52

### CODE REVIEW

File: apps/namadillo/src/App/Staking/Unstake.tsx [hunk 1]
```diff
@@ -20,18 +22,27 @@ import { useAtomValue, useSetAtom } from "jotai";
...
+import { EpochInfo, MyValidator } from "types";
 import { BondingAmountOverview } from "./BondingAmountOverview";
 import { UnstakeBondingTable } from "./UnstakeBondingTable";
 import StakingRoutes from "./routes";
 
+const getUnboundPeriod = ({
+  unbondingPeriodInEpochs,
+  minEpochDuration,
+}: EpochInfo): number => {
+  const duration = unbondingPeriodInEpochs * minEpochDuration;
```
Comment (euharrison) [inline apps/namadillo/src/App/Staking/Unstake.tsx:34]: Could someone confirm if this calculation is correct?

File: apps/namadillo/src/App/Staking/Unstake.tsx [hunk 2]
```diff
@@ -141,6 +152,9 @@ const Unstake = (): JSX.Element => {
     return "";
   })();
 
+  const unboundPeriod =
```
Comment (jurevans) [inline apps/namadillo/src/App/Staking/Unstake.tsx:155]: Small thing: could we change instances of `unboundPeriod` to `unbondPeriod` for consistency, and rename `getUboundingPeriod` to `getUnbondingPeriod`? Might be a little more clear, matching the definition in `EpochInfo`

Comment (euharrison) [reply apps/namadillo/src/App/Staking/Unstake.tsx]: ops, yes, thanks! done :)

File: apps/namadillo/src/App/Staking/Unstake.tsx (hunk 1 above)
Comment (euharrison) [reply apps/namadillo/src/App/Staking/Unstake.tsx:36]: I did some calculation these last days using and it looks correct :)

**Review by jurevans — APPROVED** [review]
LGTM!

**Pull request merged**